data_steward/bq_utils.py

```diff
@@ -739,6 +739,31 @@ def create_dataset(
     return insert_result
 
 
+def get_sql_row_expr(row: dict, fields: list):
+    """
+    Get appropriate SQL expression for a row based on fields spec
```

Comment:
nit: could you add a little more here, is this meant to be used in a SELECT statement, or is this for an insertion? opt: an example output format might be useful

Reply:
Not nitpicking- thanks for mentioning this. I will add an example.